hddfancontrol/__init__.py

```diff
@@ -259,13 +259,16 @@ def getState(self) -> DriveState:
             "sleeping": self.__class__.DriveState.SLEEPING,
         }
         cmd = ("hdparm", "-C", self.device_filepath)
-        with self.get_state_lock:
-            output = subprocess.check_output(
-                cmd, stdin=subprocess.DEVNULL, stderr=subprocess.DEVNULL, universal_newlines=True
-            )
-            self.get_state_count += 1
-        str_state = output.rsplit(" ", 1)[-1].strip()
-        state = states[str_state]
+        try:
+            with self.get_state_lock:
+                output = subprocess.check_output(
+                    cmd, stdin=subprocess.DEVNULL, stderr=subprocess.DEVNULL, universal_newlines=True
+                )
+                self.get_state_count += 1
+            str_state = output.rsplit(" ", 1)[-1].strip()
+            state = states[str_state]
+        except subprocess.CalledProcessError:
+           state = self.__class__.DriveState.UNKNOWN
```

Comment:
Please fix https://github.com/desbma/hddfancontrol/actions/runs/5847917664/job/16046081529?pr=46#step:4:147 and **test that code**. 
I can not verify it works on my system, but that indentation error would lead to a runtime error.

Reply:
Fixed.